astropy/io/ascii/ui.py

```diff
@@ -144,13 +144,13 @@ def get_reader(Reader=None, Inputter=None, Outputter=None, **kwargs):
     names : list
         List of names corresponding to each data column
     include_names : list
-        List of names to include in output (default= ``None`` selects all names)
```

Comment:
I think for these we should simply remove the description: given how the code works, everything except for the first three arguments can be omitted and thus has a default value. Since we do not give the defaults for any of the other arguments, I think we should not give them here either. If one really thinks this is important I would write `(if not given, select all names)`